mount/mount_windows.go

```diff
@@ -74,16 +75,21 @@ func (m *Mount) mount(target string) error {
 	}
 	defer func() {
 		if err != nil {
-			hcsshim.DeactivateLayer(di, layerID)
+			if layerErr := hcsshim.DeactivateLayer(di, layerID); layerErr != nil {
+				log.G(context.TODO()).WithError(layerErr).Error("failed to deactivate layer")
+			}
 		}
 	}()
 
 	if err = hcsshim.PrepareLayer(di, layerID, parentLayerPaths); err != nil {
 		return fmt.Errorf("failed to prepare layer %s: %w", m.Source, err)
 	}
+
 	defer func() {
 		if err != nil {
-			hcsshim.UnprepareLayer(di, layerID)
+			if layerErr := hcsshim.UnprepareLayer(di, layerID); layerErr != nil {
+				log.G(context.TODO()).WithError(layerErr).Error("failed to deactivate layer")
```

Comment:
Nit: "failed to unprepare layer". I'd also suggest adding "during mount failure cleanup" or something to these, so they're easily distinguishable from the same function calls failing in Unmount.